bdai_ros2_wrappers/bdai_ros2_wrappers/static_transform_broadcaster.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute Inc.  All rights reserved.
+from typing import Dict, Iterable, Optional, Union, cast
+
+from geometry_msgs.msg import TransformStamped
+from rclpy.node import Node
+from rclpy.qos import DurabilityPolicy, HistoryPolicy, QoSProfile
+from tf2_msgs.msg import TFMessage
+
+
+class StaticTransformBroadcaster:
+    """
+    A modified :class:`tf2_ros.StaticTransformBroadcaster` that stores transforms
+    sent through it, matching ``rclcpp::StaticTransformBroadcaster`` behavior.
+    """
+
+    DEFAULT_QOS = QoSProfile(depth=1, durability=DurabilityPolicy.TRANSIENT_LOCAL, history=HistoryPolicy.KEEP_LAST)
```

Comment:
Is there a reason this is kept as a class variable?

Reply:
Convenience, so that callers that may need to know what the default QoS is can read it. I'm future-proofing and it's OK if we don't too.